cmake/CmakeFunctionHelper.cmake

```diff
@@ -451,6 +451,57 @@ function(generate_blas_rotg_return_objects blas_level func)
   add_sycl_to_target(TARGET ${func} SOURCES ${FUNC_SRC})
 endfunction(generate_blas_rotg_return_objects)
 
+# blas function for generating source code for the rotg operator (asynchronous version with containers)
+function(generate_blas_rotmg_objects blas_level func)
+  set(LOCATION "${SYCLBLAS_GENERATED_SRC}/${blas_level}/${func}/")
+  foreach (executor ${executor_list})
+    foreach (data ${data_list})
+      cpp_type(cpp_data ${data})
+      set(container_list_in)
```

Comment:
I don't see where `container_list_in` gets used.

Reply:
Actually I think it should come in for y parameter which is readonly. So need to update the foreach.